### ISSUE TITLE
Context missing in Duplicate View

### ISSUE THREAD

**Issue by seanmcbreen** [Microsoft/vscode#50911]
Issue Type: <b>Bug</b>

When I use the new 'open duplicate window' some context is missing specifically break points are different in my two windows.

![image](https://user-images.githubusercontent.com/1051190/40814145-61aecd1c-64f3-11e8-82f8-17245f37e14d.png)

VS Code version: Code - Insiders 1.24.0-insider (08537497eecd3172390194693d3d7c0ec8f52b70, 2018-05-31T05:17:57.180Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6600U CPU @ 2.60GHz (4 x 2808)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: disabled_software<br>video_decode: enabled<br>video_encode: enabled<br>vpx_decode: unavailable_software<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.93GB (7.02GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code Insiders\Code - Insiders.exe|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (13)</summary>

Extension|Author (truncated)|Version
---|---|---
gitlens|eam|8.3.3
rest-client|hum|0.18.4
markdown-table-formatter|jos|0.2.3
mssql|ms-|1.3.1
python|ms-|2018.4.0
cpptools|ms-|0.17.3
java|red|0.25.0
todotasks|san|0.5.0
kustovscode|sea|0.0.4
vscode-java-debug|vsc|0.9.0
vscode-java-pack|vsc|0.3.0
vscode-java-test|vsc|0.6.1
vscode-maven|vsc|0.8.0

</details>

**Comment by seanmcbreen** [comment]
YEs I added those - what I was expecting was to see those show up in the underlying window and the ones I had set in the first Windows to also be visible.  I didn't do it but was not sure where the debugger would break on line 13 or 16.

This could be a limitation of the design and if that's the case we should document it so people understand.  Or perhaps present a notification when a user does this the first time to list out the constraints.
